roles/confluent.kafka_broker/defaults/main.yml

```diff
@@ -34,6 +34,15 @@ kafka_broker_sysctl_file: /etc/sysctl.conf
 
 kafka_broker_default_interal_replication_factor: "{{ [ groups['kafka_broker'] | length, 3 ] | min }}"
 
+kafka_broker_confluent_metrics:
+  enabled: true
+  servers: "{{groups['kafka_broker']}}"
```

Comment:
so this is a list object

Reply:
Yep, I do the join() so you don't have to (Seeing as the templated command it really long). Make it a little more flexible for people on how to pass in there brokers. Allowing the changes that where made to support multi-dcs be leverage to work within this area with minimal work on the users part. IE passing in the Kafka_broker group of choice instead of hardcoding/hand templating the join and port logic.